### DISCUSSION TITLE
Welcome to PDF DataSource for Apache Spark Discussions!

### DISCUSSION THREAD

**Discussion by mykolamelnykml** [StabRise/spark-pdf#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tkorba1** [comment]
Hello is there support for pdf files that are password protected?

**Reply by mykolamelnykml** [reply]
Hello @tkorba1. 
Not for now.

Let's discuss how to add it.
